cooltools/expected.py

```diff
@@ -987,6 +1377,287 @@ def blocksum_asymm(
     )
 
 
+# user-friendly wrapper for diagsum_symm and diagsum_pairwise - part of new "public" API
+def get_cis_expected(
+    clr,
+    view_df=None,
+    symmetric=True,
+    balance=True,
+    ignore_diags=2, # should default to cooler info
+    chunksize=10_000_000,
+    nproc=1,
+):
+    """
+    Calculate average interaction frequencies as a function of genomic
+    separation between pixels i.e. interaction decay with distance.
+    Genomic separation aka "dist" is measured in the number of bins,
+    and defined as an index of a diagonal on which pixels reside (bin1_id - bin2_id).
+
+    Average values are reported in the columns with names {}.avg, and they
+    are calculated as a ratio between a corresponding sum {}.sum and the
+    total number of "valid" pixels on the diagonal "n_valid".
+
+
+    Parameters
+    ----------
+    clr : cooler.Cooler
+        Cooler object
+    view_df : viewframe
+        a collection of genomic intervals where expected is calculated
+        otherwise expected is calculated for full chromosomes.
+    symmetric: bool
+        When True, view_df defines symmetric on-diagonal regions,
+        otherwise expected is calculated for every intra-chromosomal
+        combination of regions in view_df
+    balance : bool or str
```

Comment:
turn it into `clr_weight_name` that is `weight` by default. And `clr_weight_name=None` would do the unbalanced ...